worker/Cargo.toml

```diff
@@ -98,6 +98,11 @@ git = "https://github.com/paritytech/substrate.git"
 branch = "master"
 version = "3.0.0"
 
+[dependencies.frame-support]
+git = "https://github.com/paritytech/substrate.git"
+branch = "master"
+version = "3.0.0"
```

Comment:
frame-support as regular dependency, moved from the `dev-dependencies` section below